CROSSSECTOR/IT/CloudRegion/examples/example-normalized.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "urn:ngsi-ld:CloudRegion:id:Noida",
+  "type": "CloudRegion",
+  "name": {
+    "type": "Text",
+    "value": "Noida"
+  },
+  "contactPoint": {
```

Comment:
it is not following the structure of a contact point (coming from schema.org), Better to change name / structure

Reply:
see bellow.